client/src/App.js

```diff
@@ -3,23 +3,51 @@ import ChatPage from './pages/ChatPage.js';
 import LoginPage from "./pages/LoginPage.js";
 import "./App.css";
 import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
+import '../node_modules/bootstrap/dist/js/bootstrap.bundle.min.js';
+
+const url = "http://localhost:5000/";
+//fetch(url + "login/" + localStorage.getItem('login')))
 
 class App extends Component {
+    componentDidMount(){
+        var id;
+        //Verify that a valid login is saved
+        if((id = localStorage.getItem('login'))!==''){
+            (async () => {
+                let response;
+                try{
+                    response = await fetch(url + "user/login/" + id)
+                } catch (ex) {
+                    console.log("Error", response.status);
+                }
+                if(response.ok){
+                    var json = response.json();
+                    if(json._id === id){
+                        this.setState({loggedIn: true});
+                    }
+                } else {
+                    console.log("Error", response.status);
+                }
+            })()
+        }
+    }
+
     constructor(props) {
         super(props);
         this.state = {
             apiResponse: "",
-            loggedIn: localStorage.getItem('loggedIn') ? JSON.parse(localStorage.getItem('loggedIn')) : false ,
+            loggedIn: false,
             loginError: "",
             email: "",
             messages: [{text: "test message"}]
         };
     }
-
-    onSuccessfulLogin(email){
-        this.setState({loggedIn: true})
-        localStorage.setItem('loggedIn',JSON.stringify(this.state.loggedIn))
-        this.setState({email: email})
+    updateLogin(loginSuccessful){
```

Comment:
To be consistent with the other names, can we call this handleLogin?

Reply:
This has been changed